### ISSUE TITLE
Allow the user to specify a local schema

### ISSUE THREAD

**Issue by edeutsch** [NCATSTranslator/reasoner-validator#81]
It would be very useful to allow the user to specify a local schema file. The pull-automatically-from-GitHub is a fine idea, and good for many users/use cases, but has some drawbacks:

1) If you invoke the validator "too often", GitHub will lock you out for a while. Their API only allows N calls per hour or something, and if you exceed that while testing, you have to wait.

2) It makes it very awkward to test schema fixes. I am struggling with a suspected problem in TRAPI 1.4.0-beta4, and it would be very nice to be able to validate against a local yaml file that I can tweak to make sure I have the fix correct before submitting a PR to the TRAPI repo.

I'm envisioning this:
```
validator = TRAPIResponseValidator(trapi_schema='./TranslatorReasonerAPI_experimental.yaml', biolink_version="3.2.1")
```

**Comment by RichardBruskiewich** [comment]
@edeutsch, as I noted on Slack, I'd be interested in simply overloading (and documenting revisions to) the existing **`trapi_version`** string parameter found throughout the code base, to also accept a YAML file name or path, that is, 

```
validator = TRAPIResponseValidator(trapi_version='./TranslatorReasonerAPI_experimental.yaml', biolink_version="3.2.1")
```

Although potentially slightly confusing, it may be actually simple to implement without too much extra overhead.

**Comment by edeutsch** [comment]
sure, that's fine with me

**Comment by RichardBruskiewich** [comment]
Implemented in release v3.5.4
